x/wasm/keeper/querier_test.go

```diff
@@ -708,17 +722,22 @@ func TestQueryPinnedCodes(t *testing.T) {
 			expCodeIDs: []uint64{exampleContract2.CodeID},
 		},
 	}
-	for msg, spec := range specs {
-		t.Run(msg, func(t *testing.T) {
+	for testName, spec := range specs {
+		t.Run(testName, func(t *testing.T) {
```

Comment:
is this change needed?

Reply:
Unnecessary change. Revert changes. Thanks.